src/org/safs/rest/service/models/consumers/SafsrestAdapter.groovy

```diff
@@ -144,15 +143,11 @@ class SafsrestAdapter extends ClientPOJOAdapter {
         if (request.headers) {
             request.headers.each { entry -> optionList << /${CurlCommand.HEADER_OPTION}${entry.key}${CurlCommand.HEADER_FIELD_SEPARATOR}${entry.value}/ }
         }
-
-		if(auth2!=null /* TODO In what condition, we need to make request to authorization server. */){
```

Comment:
Actually, I see now what is being done here.  The auth server info is put in just before the token is obtained, then after the token, the entrypoint's info is used.  I'll restore that functionality.